src/pages-conditional/what-is-ethereum.tsx

```diff
@@ -374,141 +405,694 @@ const WhatIsEthereumPage = ({
+              </p>
+              <p>
+                Every action on the Ethereum network requires a certain amount
+                of computational power. This is paid in the form of Ether fee.
+                This means you need at least small amount of ETH to use the
+                network.
+              </p>
+              <p>
+                <ButtonRow>
+                  <ButtonLink to="/eth/">What is Ether?</ButtonLink>
+                  <ButtonLink to="/get-eth/" isSecondary>
+                    Get ETH
+                  </ButtonLink>
+                </ButtonRow>
+              </p>
+            </Width60>
+          </TwoColumnContent>
+        </Section>
+
+        <Section>
+          <TwoColumnContent reverse>
+            <Width40>
+              <GatsbyImage image={getImage(data.meetEth)} />
+            </Width40>
+            <Width60>
+              <h2>What can I do with ETH coin?</h2>
+              <p>
+                Turns out: a lot of things! One of the most prominent usages of
+                the Ethereum technology is decentralised finance (DeFi) that
+                opens whole area of banking services to anybody with an internet
+                connection. You can use your Ether as a collateral to take out
+                loans or provide liquidity to earn interest on your funds.
+              </p>
+              <p>
+                <ButtonRow>
+                  <ButtonLink to="/dapps/">Explore applications</ButtonLink>
+                  <ButtonLink to="/defi/" isSecondary>
+                    Learn about DeFi
+                  </ButtonLink>
+                </ButtonRow>
+              </p>
+            </Width60>
+          </TwoColumnContent>
+        </Section>
+
+        <Section bgColor={theme.colors.homeBoxPurple}>
+          <TwoColumnContent>
+            <Width40>
+              <GatsbyImage image={getImage(data.whoRunsEthereum)} />
+            </Width40>
+            <Width60>
+              <h2>Who runs Ethereum?</h2>
+              <p>
+                Ethereum is not controlled by any one entity. It exists solely
+                through the decentralised participation and cooperation of the
+                community. Ethereum makes use of nodes (a computer with a copy
+                of the Ethereum blockchain data) run by volunteers to replace
+                individual server and cloud systems owned by major internet
+                providers and services.
+              </p>
+              <p>
+                These distributed nodes, run by individuals and businesses all
+                over the world, provide resiliency to the Ethereum network
+                infrastructure. It is therefore much less vulnerable to hacks or
+                shutdowns.{" "}
+                <b>
+                  Since its launch in 2015, Ethereum has never suffered downtime
+                </b>
+                . There are currently over 1777 individual nodes running
```

Comment:
This number (1777) should be pulled in dynamically via the Etherscan API.

Reply:
We have changed the copy here. Updated